apps/src/templates/sectionProgressV2/LevelDataCell.jsx

```diff
@@ -32,14 +34,40 @@ export default function LevelDataCell({
   studentId,
   sectionId,
   studentLevelProgress,
-  overrideIcon,
+  expandedChoiceLevel,
 }) {
-  const levelData = React.useMemo(() => {
-    if (!studentLevelProgress) {
+  const itemType = React.useMemo(() => {
```

Comment:
Doesn't have to be this PR, but might be nice to have tests for each of these cases.